spec/system/admin/facility_orders_search_spec.rb

```diff
@@ -99,12 +99,12 @@
           expect(page).to have_css(".fa-users", count: 1) # cross_core_orders.last
           expect(page).to have_css(".fa-building", count: 1) # cross_core_order_originating_facility
 
-          expect(page).to have_content(cross_core_order_originating_facility.id)
-          expect(page).to have_content(cross_core_orders.last.id)
+          expect(page).to have_link(cross_core_order_originating_facility.id, href: facility_order_path(cross_core_order_originating_facility.facility, cross_core_order_originating_facility))
```

Comment:
Were you able to figure out why this was failing?  Why does specifying the href fix it?

Maybe we can set these expected hrefs in variable to make this more readable?
Also, `cross_core_order_originating_facility` is an order, so how about we call it `originating_cross_core_order`?